src/Aspire.Hosting.AWS/Lambda/LambdaExtensions.cs

```diff
@@ -97,6 +96,33 @@ public static class LambdaExtensions
         return resource;
     }
 
+    /// <summary>
+    /// This method is adapted from the Aspire WithProjectDefaults method.
+    /// https://github.com/dotnet/aspire/blob/157f312e39300912b37a14f59beda217c8195e14/src/Aspire.Hosting/ProjectResourceBuilderExtensions.cs#L287
+    /// </summary>
+    /// <param name="builder"></param>
+    /// <returns></returns>
+    private static IResourceBuilder<LambdaProjectResource> WithOpenTelemetry(this IResourceBuilder<LambdaProjectResource> builder)
+    {
+        builder.WithEnvironment("OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EXCEPTION_LOG_ATTRIBUTES", "true");
+        builder.WithEnvironment("OTEL_DOTNET_EXPERIMENTAL_OTLP_EMIT_EVENT_LOG_ATTRIBUTES", "true");
+        // .NET SDK has experimental support for retries. Enable with env var.
+        // https://github.com/open-telemetry/opentelemetry-dotnet/pull/5495
```

Comment:
This PR is merged, is the support for retries still experimental?

Reply:
Oh, I see that's the PR for the environment variable 🤦

It's still experimental according to this: https://github.com/vishweshbankwar/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Exporter.OpenTelemetryProtocol/README.md#experimental-features